Assets/Scripts/DeliveryExperiment.cs

```diff
@@ -50,7 +50,9 @@ public class DeliveryExperiment : CoroutineExperiment
     // TODO: JPB: Make these configuration variables
 
     // Experiment type
-    private const bool HOSPITAL_COURIER = true;
+    // LC: TODO: maybe set up a real "standard" courier?
+    private const bool STANDARD_COURIER = false;
```

Comment:
Should we remove STANDARD_COURIER?

Reply:
My intent was to create a "basic" courier version, but since we are recreating a whole new courier, we can just remove this :)